sdk/identity/identity/CHANGELOG.md

```diff
@@ -1,20 +1,98 @@
 # Release History
 
-## 2.0.0-beta.7 (Unreleased)
+## 2.0.0 (2021-10-12)
+
+We're proud to announce the first GA release of the `@azure/identity` v2 package. The v2 package includes the best parts of v1, plus several improvements.
+
+This changelog entry showcases the changes that have been made. For a v1-to-v2 migration guide, see the [migration guide](https://github.com/Azure/azure-sdk-for-js/blob/main/sdk/identity/identity/migration-v1-v2.md).
 
 ### Features Added
 
+#### Plugin API
+
+Identity v2 provides a top-level `useIdentityPlugin` method, which allows using two new plugin packages:
+
+- [@azure/identity-vscode](https://www.npmjs.com/package/@azure/identity-vscode), which provides the dependencies of `VisualStudioCodeCredential` and enables it.
+- [@azure/identity-cache-persistence](https://www.npmjs.com/package/@azure/identity-cache-persistence), which provides persistent token caching.
+- If the `@azure/identity-vscode` plugin isn't used through the `useIdentityPlugin` function, the `VisualStudioCodeCredential` exposed by Identity v2 will throw a `CredentialUnavailableError`.
+
+Most credentials on Identity v2 now include a constructor option called `tokenCachePersistenceOptions`. If set to `tokenCachePersistenceOptions: { enabled: true }`, it will enable persistent token caching on that credential instance.
```

Comment:
Since we are setting this to `{ enabled: true }` and not just `true`, it might be worth calling out that there are other things that can be configured for the persistent cache features. It would also be useful to point from here to a place where we talk a bit more about the persistent cache feature and how it works.

```suggestion
Most credentials on Identity v2 now support the persistent token caching feature. Such credentials include the property [tokenCachePersistenceOptions](https://docs.microsoft.com/javascript/api/@azure/identity/tokencachepersistenceoptions) in the constructor options which can be used to enable this feature.
```

Additionally, you can add the sample code here as an example. Just because there is a plugin involved.

```js
import { cachePersistencePlugin } from "@azure/identity-cache-persistence";
import { useIdentityPlugin, DeviceCodeCredential } from "@azure/identity";

useIdentityPlugin(cachePersistencePlugin);

async function main() {
  const credential = new DeviceCodeCredential({
    tokenCachePersistenceOptions: {
      enabled: true
    }
  });
}
```

Reply:
Let me know how it looks now!